roycemoses/cs179n-project
# Aavit004/inventory

To test Pause Menu:
In sprint4 scene make sure the inventory canvas is hid.
Play game and press esc to open and close pause menu. 
click on the resume button to resume
click on the exit button to exit (this will later be linked to the MENUGB scene)
